docs/api/websocket.md

```diff
@@ -368,11 +368,14 @@ The server will indicate with a message indicating that the service is done exec
       "id": "326ef27d19415c60c492fe330945f954",
       "parent_id": null,
       "user_id": "31ddb597e03147118cf8d2f8fbea5553"
-    }
+    },
+    "response": null
   }
 }
 ```
 
+If the service supports providing return data, it will automatically be included in the `result`.
```

Comment:
We should mention the `response` key and clarify that it will always be present and have a `null` value if no service response data is present and the response data if the service supports response value.